specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-03-31/cdn.json

```diff
@@ -4282,6 +4343,62 @@
           }
         }
       },
+      "UrlSigningKey": {
+        "description": "Url signing key",
+        "required": [
+          "keyId",
+          "keySourceParameters"
+        ],
+        "properties": {
+          "keyId": {
+            "description": "Defines the customer defined key Id. This id will exist in the incoming request to indicate the key used to form the hash.",
+            "type": "string"
+          },
+          "keySourceParameters": {
+            "description": "Defines the parameters for using customer key vault for Url Signing Key.",
+            "$ref": "#/definitions/KeyVaultSigningKeyParameters"
+          }
+        }
+      },
+      "KeyVaultSigningKeyParameters": {
+        "description": "Describes the parameters for using a user's KeyVault for URL Signing Key.",
+        "required": [
+          "@odata.type",
+          "subscriptionId",
+          "resourceGroupName",
+          "vaultName",
+          "secretName",
+          "secretVersion"
+        ],
+        "properties": {
+          "@odata.type": {
+            "type": "string",
+            "enum": [
+              "#Microsoft.Azure.Cdn.Models.KeyVaultCertificateSourceParameters"
```

Comment:
This should be KeyVaultSigningKeyParameters

Reply:
Fixed